### PR TITLE
DOCS-611: Change to by-id on webcam model

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by sguequierre — APPROVED** [review]
LGTM one comment

File: docs/components/camera/webcam.md [hunk 1]
```diff
@@ -101,12 +101,17 @@ To list available `video_path`s use the following command:
...
+
+To find the `path`s of all connected video devices, run the following command:
+
 ```sh
 v4l2-ctl --list-devices
 ```
 
-The name of the device, listed after `/dev/` is the `video_path`.
-For example, if your device is at `/dev/video0`, your `video_path` is `video0`.
+The `id` is a more consistent way to refer to the webcam.
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/components/camera/webcam.md:114]: The first command lists all the ids - the second one all the paths

Comment (npentrel) [inline docs/components/camera/webcam.md:114]:
```suggestion
The `id` listed from `ls /dev/v4l/by-id/` is a more consistent way to refer to the webcam.
```

File: docs/components/camera/webcam.md [hunk 2]
```diff
@@ -101,12 +101,17 @@ To list available `video_path`s use the following command:
...
+
+To find the `path`s of all connected video devices, run the following command:
+
 ```sh
 v4l2-ctl --list-devices
 ```
 
-The name of the device, listed after `/dev/` is the `video_path`.
-For example, if your device is at `/dev/video0`, your `video_path` is `video0`.
+The `id` listed from `ls /dev/v4l/by-id/` is a more consistent way to refer to the webcam.
```
Comment (npentrel) [inline docs/components/camera/webcam.md:114]:
```suggestion
The `id` listed by `ls /dev/v4l/by-id/` is a more consistent way to refer to the webcam.
```

**Pull request merged**